src/components/BookingForm.vue

```diff
@@ -142,16 +148,31 @@ export default {
       const availabilities = await getAsync(url);
       this.availabilities = availabilities.data;
     },
+    async fetchBookingsList() {
+      const bookings = await getAsync(`/bookings?date=${this.bookingDate}&office=${this.selectedOffice.id}`)
+
+      // The Email is temporarily used as a username
+      // In the future real usernames and profile pictures will be supported
+      const extractUsername = (booking) => {
+        const [username] = booking.user.email.split("@");
+        return username;
+      };
+
+      this.bookings = bookings.data.items.map(extractUsername)
```

Comment:
Generally, when writing code, you should try to follow what's called the "Single Responsibility Principle". Essentially, each class / module / vue should have a single thing that they do. In this case, the logic for choosing what information from a booking should be displayed belongs, in my opinion, in the BadUserListItem.

BookingForm takes care of getting the data when the availability date is chosen. Then the BadUserList's responsibility is to hold the list of BadUserListItem. BadUserListItem then chooses from the Booking what information they want to display and how.

This has the following advantage:
Let's say we add user pictures to the bookings. With my suggested change you would only need to change BadUserListItem to handle the new picture. Instead of now, where you would have to change BookingForm, BadUserList and BadUserListItem